code/modules/food/recipes_microwave.dm

```diff
@@ -1400,6 +1400,32 @@
 	reagents = list("milk" = 15, "sodiumchloride" = 5)
 	result = /obj/item/weapon/reagent_containers/food/snacks/gigapuddi/anger
 
+//LIVING PUDDI
+//This is a terrible idea.
+
+/datum/recipe/livingpuddi
```

Comment:
This has to go at the end of this section.
Due to the way the recipes are generated and checked in the code, the first good recipe (this one) will be selected for all the ingredient combinations, so the sub-recipes won't complete.